### ISSUE TITLE
[Bug][a11y]: Thumbnail selected state not using Highlight for border color with forced-colors: active 

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3405]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Thumbnail

### Expected behavior

<img width="999" alt="Screen shot showing Thumbnail with fixes WHCM" src="https://github.com/adobe/spectrum-web-components/assets/154077/d2e13414-6dec-4321-9a6e-d383d2ae5bbe">

```css
@media (forced-colors: active) {
    :host {
        --spectrum-thumbnail-layer-border-color-outer: CanvasText;
        --spectrum-thumbnail-layer-border-color-inner: Canvas;
        --highcontrast-thumbnail-border-color-selected: Highlight;
    }
}
```

### Actual behavior

The selected Thumbnail state border color does not render using the Highlight color to indicate selection with forced-colors: active.

Notice in the following screen shot how the .layer-inner has outline color remains white, rather than becoming the Canvas color, and the selected style does not use Highlight color with forced-colors: active.

<img width="999" alt="Screen shot showing Thumbnail in WHCM" src="https://github.com/adobe/spectrum-web-components/assets/154077/8da67d9c-b9af-45fb-aa63-ebd6dffb2ce2">

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error

**Comment by najikahalsema** [comment]
@pfulton Can you confirm this in CSS for us?

**Comment by pfulton** [comment]
> @pfulton Can you confirm this in CSS for us?

Yes, it looks like this is incorrect on our end. Here's how ours renders in `forced-colors`

<img width="691" alt="Screenshot 2023-10-26 at 1 16 30 PM" src="https://github.com/adobe/spectrum-web-components/assets/360251/cf8ad396-aad7-4923-bdde-a0cc3abd3f9d">

**Comment by Westbrook** [comment]
Are you sure that's `Highlight` (#1eebff ish) as opposed to `Spectrum blue` (#2772d8 ish)?

**Comment by pfulton** [comment]
To clarify my comment above: it appears that Spectrum CSS hasn't addressed this specific usage at all (hence why I think we see the spectrum blue-ish color), and that it's incorrect on our end, so not just happening within SWC, but also in CSS.

I'll get a ticket in our backlog to address this and get the correct `forced-colors` experience implemented.

**Comment by Westbrook** [comment]
🙈 my bad, misread your comment. Thanks for getting this ticketed! 🙇🏼

**Comment by najikahalsema** [comment]
fixed via spectrum-css/#2254
